fragment/fragment/src/main/java/androidx/fragment/app/FragmentActivity.java

```diff
@@ -645,6 +649,19 @@ public void startActivityFromFragment(@NonNull Fragment fragment,
         fragment.startActivityForResult(intent, requestCode, options);
     }
 
+    @Override
+    public void reportFullyDrawn() {
+        // On KitKat (API 19) the Activity.reportFullyDrawn() method requires the
+        // UPDATE_DEVICE_STATS permission, otherwise it throws an exception. Instead of throwing,
+        // we fall back to a no-op call here.
+        if (Build.VERSION.SDK_INT == Build.VERSION_CODES.KITKAT && ContextCompat
```

Comment:
The general convention in AndroidX code is to use the numbers themselves, rather than `Build.VERSION_CODES` values. Please use 19 here.